test/unit/translators/twitter_translators/user_translator_test.rb

```diff
@@ -1,18 +1,18 @@
-require "test_helper"
+require 'test_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.